packages/example-app-base/tslint.json

```diff
@@ -2,6 +2,6 @@
   "extends": ["@uifabric/tslint-rules"],
   "rules": {
     "deprecation": false,
-    "prefer-const": false
+    "typedef": false
```

Comment:
any reason adding this?

Reply:
She's removing line 57 of `ExampleCard.tsx`:

`// tslint:disable-next-line:typedef`

Reply:
Most of the other projects disable the typedef rule. I agree with disabling it because I like the ability to leave the types off in some cases where it can easily be inferred, for example in map/filter callbacks. Now that VS Code's intellisense is pretty reliable, you can hover over the thing if you want to see the type. (Also, since I enabled noImplicitAny, specifying the type is still required in any place where there's ambiguity.)
```ts
const foo: SomeReallyComplicatedType[] = [/*stuff*/];
const bar = foo.map(thing => doStuffWithThing(thing));
```

Reply:
@dzearing  I'd like to merge this ASAP since it's the first in a chain of changes. If you want the typedef change reverted, let me know.